cluster/examples/kubernetes/ceph/admission-controller/webhook-config.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: ${WEBHOOK_CONFIG_NAME}
+  namespace: ${NAMESPACE}
+webhooks:
+  - name: ${SERVICE_NAME}.${NAMESPACE}.svc
+    rules:
+      - apiGroups:   ["ceph.rook.io"]
+        apiVersions: ["v1"]
+        operations:  ["CREATE","UPDATE","DELETE"]
+        resources:   ["*"]
+    clientConfig:
+      service:
+        name: ${SERVICE_NAME}
+        namespace: ${NAMESPACE}
+        path: /validate
+      caBundle: ${CA_BUNDLE}
+    admissionReviewVersions: ["v1beta1"]
+    sideEffects: None
+    timeoutSeconds: 5
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Actually let's add the service account, clusterrole, and clusterrolebinding to the common.yaml. I know we had some discussion previously about just having them in this script, but here are the reasons for adding them to common.yaml:
- The permissions granted to the service account are a small subset of the permissions the operator is already requesting. 
- Those resources are already added to the helm chart and the integration tests. We really should have consistency between common.yaml and the helm chart.
- The ValidatingWebhookConfiguration resource is really the only resource we need to be careful creating only when needed.